doc/R.nvim.txt

```diff
@@ -2115,7 +2144,8 @@ mapped in the current session, do in Normal mode:
 >vim
    :RMapsDesc
 <
-The prefix "RD" in the labels means "cursor down".
+You may close the window opened by `:RMapsDesc` by simply pressing `q`. The
+prefix "RD" in the labels means "cursor down".
```

Comment:
Do you want to mean that in all the send commands? Should it be placed somewhere else that is better related to the sending commands.

Reply:
Yes, the prefix "R" is in all labels, and the letter "D" in "RD" means "Down" and is in many sending commands.  However, users will only need to know this if they are willing to customize the key bindings. So, I think section _8. Custom key bindings_ is the proper place for this information.